compose/compose-file/index.md

```diff
@@ -348,33 +348,12 @@ Simple example:
 > for a service to be ready, see [Controlling startup order](/compose/startup-order.md)
 > for more on this problem and strategies for solving it.
 
-A healthcheck indicates you want a dependency to wait
-for another container to be "healthy" (i.e. its healthcheck advertises a
-successful state) before starting.
+> **NOTE:** version 3 no longer supports the `condition` form of `depends_on`
```

Comment:
@dnephin I think this note can be moved to https://docs.docker.com/compose/compose-file/compose-versioning/#version-3